FlightReservation/App.js

```diff
@@ -6,21 +6,21 @@
  * @flow strict-local
  */
 
-import React, { useState } from 'react';
-import {
-  SafeAreaView,
-  StatusBar,
-} from 'react-native';
-
+import React, {useState} from 'react';
+import {SafeAreaView, StatusBar} from 'react-native';
 import MainScreen from './screens/mainScreen';
+import MyFlights from './screens/myFlights';
+import BookingScreen from './screens/bookingScreen';
 
 const App: () => React$Node = () => {
-  
+
   return (
     <>
-      <StatusBar barStyle="dark-content" backgroundColor="white"/>
+      <StatusBar barStyle="dark-content" backgroundColor="white" />
       <SafeAreaView>
-        <MainScreen/>
+        <MyFlights />
+        {/* <MainScreen /> */}
+        {/* <BookingScreen /> */}
```

Comment:
```suggestion
```

Reply:
We put it like this because we needed to move between screens, in the branch sand-box we already implemented a stack react-navigation